x/gov/migrations/v3/convert_test.go

```diff
@@ -46,7 +46,7 @@ func TestConvertToLegacyProposal(t *testing.T) {
 
 	for name, tc := range testCases {
 		t.Run(name, func(t *testing.T) {
-			proposal.FinalTallyResult = &tc.tallyResult
+			proposal.FinalTallyResult = &tc.tallyResult //nolint:exportloopref
```

Comment:
Instead of doing that, don't we just need to re-assign as `tc := tc` in the loop?

Reply:
didn't know we could do this!  nice!